testcases/connectivity/Serial/serial.robot

```diff
@@ -30,121 +37,95 @@ Serial Command
     sleep   1s
     [Return]  ${string}
 
-Close Connection
-    disconnect      ${TEST_CONNECTION}
+Release Test Environment
+    disconnect      ${CONNECTION_NAME}
 
-Get COM Status
-    [Arguments]  ${com_name}
-    ${conf}=    Create Dictionary   port=${com_name}
-    ${level}=   Set Log Level   NONE
-    ${dummy}    ${msg}=     Run Keyword And Ignore Error
-    ...     connect     conn_name=check_status
-    ...                 conn_type=SerialClient
-    ...                 conn_conf=${conf}
-
-    ${match}    ${dummy}=    Run Keyword And Ignore Error    Should Contain  ${msg}    PermissionError(13, 'Access is denied.', None, 5)
-    Set Log Level   ${level}
-    ${status}=  Set Variable If     '${match}' == 'PASS'    Unavailable   Unknown
-#    ${status}=  Set Variable If     ${msg}    Available   ${status}
-    Run keyword if  '${status}' == 'Available'    disconnect  check_status
-    [Return]    ${status}
+Serial Client Setup
+    testsuites.testsuite_setup    ../../../config/testsuites_config.json
+    Skip If    not ${__TESTBENCH__CONFIG}[sw][has_serial]    Skip SSH suite
```

Comment:
Hi Thomas,

It will create the **Skip** state for all tests, They are still logged in report/log files (with light yellow color).

Thank you,
Ngoan

Reply:
Hi Ngoan,
hm... this does not make me happy. Then we still have log entries, despite we configured that we don't want to test...
Is there no better way to select executed tests based on parameter configurations in robot framework?
If there is really no way, then we need to check if we can do preprocessing similar to TML, too.
Thank you,
 Thomas